rotkehlchen/tests/fixtures/rotkehlchen.py

```diff
@@ -356,10 +357,27 @@ def fixture_rotkehlchen_api_server(
         current_price_oracles_order=current_price_oracles_order,
     )
     if start_with_logged_in_user and network_mocking:
-        with patch_avalanche_request(
-                api_server.rest_api.rotkehlchen.chains_aggregator.avalanche,
-                web3_mock_data,
-        ):
+        patch_avalanche = patch_avalanche_request(
+            api_server.rest_api.rotkehlchen.chains_aggregator.avalanche,
+            web3_mock_data,
+        )
+
+        with ExitStack() as stack:
+            stack.enter_context(patch_avalanche)
+            if mock_other_web3 is True:
+                ethereum_inquirer = api_server.rest_api.rotkehlchen.chains_aggregator.ethereum.node_inquirer  # noqa: E501
+
+                wait_until_all_nodes_connected(
```

Comment:
Why are you doing this herE? This is already happening in `initialize_mock_rotkehlchen_instance`

Reply:
yes, it happens inside the `initialize_mock_rotkehlchen_instance`. Didn't see it. Thanks